libs/go/aealite/README.md

```diff
@@ -19,37 +29,34 @@ func main() {
 
 	var err error
 
-	logger.Info().Msg("Agent starting ...")
+	// env file
+	if len(os.Args) != 2 {
+		log.Print("Usage: main ENV_FILE")
+		os.Exit(1)
+	}
+	envFile := os.Args[1]
+
+	log.Print("Agent starting ...")
+
 
 	// Create agent
 	agent := aea.Agent{}
 
 	// Set connection
-	agent.connection = &connections.P2PClientApi{}
+	agent.Connection = &connections.P2PClientApi{}
 
 	// Initialise agent from environment file (first arg to process)
-	err = agent.InitFromEnv()
+	err = agent.InitFromEnv(envFile)
```

Comment:
`envFile` is passed as argument to the function rather than loading it from `os.Args` inside the method